swarm/src/test/resources/docker/swarm.tests.bom

```diff
@@ -396,6 +396,7 @@ brooklyn.catalog:
         swarm.port: 4000
         docker.recovery.stabilizationDelay: 10s
         docker.recovery.failOnRecurringFailuresInThisDuration: 5m
+        swarm.sharedsecuritygroup.create: true
```

Comment:
But the point is that on some clouds this default will not work, the user _must_ make a change from the default settings. Or are there going to be specific configurations for those clouds with this set to `false` as well?